spec/features/users/user_login_spec.rb

```diff
@@ -295,5 +295,24 @@
       end
 
     end
+
+    context 'when both Earthdata Login and Launchpad Login requirements are turned off', js: true do
```

Comment:
shouldn't we have this test for both MMT and dMMT?

Reply:
also maybe check for the buttons?

Reply:
The test already existed for MMT.  Updated both tests to have modern expectations.  They are now using the have_no_link version of what other tests use to confirm the presence of the buttons.